config/configuration.toml.in

```diff
@@ -96,6 +96,8 @@ path = "@SHIMPATH@"
 # (default: disabled)
 #enable_debug = true
 
+[agent.cc]
```

Comment:
I can see that this was missing from the original file, but I'm not clear why it's being added now (particularly since this PR is specifically for kata). A couple of questions:

- Are there any legitimate values for this section for CC?
- Do we envisage an `agent.kata` section with options?

If we really should have this, could you add it on a separate commit (and maybe add a comment explaining why we have this as it might confuse users [since the agen't isn't particularly visible to them :smile:]).

Reply:
Yes, this was omitted until now because it was not needed to specify the type of agent. Indeed, the default was to use `cc`. And because we don't have any specific fields under this section, there was no point in adding it.
Now that we enable the possibility to use kata, we introduce this field to the configuration.toml file so that a user can easily understand how to modify the agent from `cc` to `kata`. That's why I think this should stay into the same commit.

Reply:
Oooh - now I understand. But how about adding a comment to explain that? Something like:

```
# This "nop" section (which doesn't accept any options currently) exists
# to make it clear that the default agent type is for Clear Containers.
[agent.cc]
```